### PR TITLE
Fix document for Kindle and add description about kindlegen

### PR DESCRIPTION
**Pull request by shibukawa**

### CODE REVIEW

File: doc/faq.rst [hunk 1]
```diff
@@ -167,11 +167,45 @@ The following list gives some hints for the creation of epub files:
...
+  for Kindle. You can get ``yourdoc.mobi`` under ``_build/epub`` after the
+  following command:
+
+  .. code-block:: bash
+
+     $ make epub
+     $ kindlegen _build/epub/yourdoc.epub
+
+  kindlegen commands doesn't accept documents that have section
+  titiles surrounding ``toctree`` directive:
```
Comment (tk0miya) [inline doc/faq.rst]: maybe typo: titles

**Comment by tk0miya** [comment]
LGTM with nits

**Comment by shibukawa** [comment]
fix

**Comment by tk0miya** [comment]
Thank you always!

**Pull request merged**